models/Asset/Service.php

```diff
@@ -217,13 +220,28 @@ public static function gridAssetData($asset, $fields = null, $requestedLanguage
                 } else {
                     if (isset($fieldDef[1])) {
                         $language = ($fieldDef[1] === 'none' ? '' : $fieldDef[1]);
-                        $metaData = $asset->getMetadata($fieldDef[0], $language, true);
+                        $rawMetaData = $asset->getMetadata($fieldDef[0], $language, true, true);
                     } else {
-                        $metaData = $asset->getMetadata($field, $requestedLanguage, true);
+                        $rawMetaData = $asset->getMetadata($field, $requestedLanguage, true, true);
                     }
 
-                    if ($metaData instanceof Model\Element\AbstractElement) {
-                        $metaData = $metaData->getFullPath();
+                    $metaData = $rawMetaData["data"] ?? null;
```

Comment:
Elements Id shown instead of path on metadata grid
![image](https://user-images.githubusercontent.com/5137917/88269475-5fc37700-ccd4-11ea-81d2-e160c60a713b.png)